### PR TITLE
Removes ++ and -- in conditionals

### PR DESCRIPTION
**Pull request by Time-Green**
I hate this and apparently so does @Mothblocks 

See #71887 

no changelog

(There's also a few in main.dm but I prefer not to touch it since most of that is optimized code)

### CODE REVIEW

**Comment by Time-Green** [comment]
three seconds I based it on the wrong branch

**Review by MrMelbert — CHANGES_REQUESTED** [review]

File: code/modules/hallucination/battle.dm [hunk 1]
```diff
@@ -47,7 +47,8 @@
 		qdel(src)
 
 	// Or, if we do have shots left, keep it going.
-	else if(--shots_left > 0)
+	else if(shots_left >= 0)
+		shots_left++
```
[the remark this answers is not in the document]
Comment (Time-Green) [reply code/modules/hallucination/battle.dm:51]: Fuck I meant --
Good catch cause this would've been not fun to fix

**Comment by Autisem** [comment]
*rants about how technicaly  prefixes are more optimized because they have 1-2 less lines of assembly code*

meme

**Comment by jlsnow301** [comment]
Accessibility over efficiency

**Pull request merged**